detox/specs/elements/ActivityIndicator.ts

```diff
@@ -1,4 +0,0 @@
-import { by, element } from "detox";
```

Comment:
Are the "helpers" in `./detox/specs/elements/` and specs in `./detox/specs/` not used?

Reply:
Not as far as I could see. Pratik mentioned he also thought these tests did not run. He used them mostly as inspiration.